diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/BlockStructureBraces.kt

```diff
@@ -144,6 +152,9 @@ class BlockStructureBraces : Rule("block-structure") {
         }
     }
 
+    private fun isNodeEmpty(node: ASTNode?) = listOf(LBRACE, RBRACE, WHITE_SPACE)
+            .containsAll(node?.getChildren(null)?.distinct()?.map { it.elementType } ?: emptyList())
```

Comment:
Several questions on this method:
* I suggest to rename it to `isBlockEmpty` since it should check only `BLOCK` nodes
* If you need to receive nullable type here, I suggest not checking for `emptyList` but explicitly returning boolean value if argument is `null`
* Why do you need `distinct` on children? I guess white space between braces is always represented by single node.
* Previously you had comparison of `children.size` with `4`, and now there is a list of tree element types. Why is it so?
* To make comparison more explicit and depending on order of nodes, instead of calling `containsAll` I suggest to zip two lists and compare elements.